x-pack/plugin/build.gradle

```diff
@@ -109,3 +110,15 @@ testClusters.integTest {
   extraConfigFile nodeCert.name, nodeCert
   extraConfigFile 'roles.yml', file('src/test/resources/roles.yml')
 }
+
+validateRestSpec {
+  ignore 'searchable_snapshots.mount.json'
```

Comment:
What's the reason to ignore these files? Do they fail validation? If so, shouldn't we fix the failures so it doesn't break the clients team?

Reply:
yeah, these will need to be fixed. I plan to log issues to fix once this get merged.

Reply:
Once we get this fixed I'd argue for _removing_ the ability to ignore. The purpose of doing validation here is so that the clients folks don't have to do it downstream. Not much point invalidating these twice on both the producer and consumer sides.

Going forward we need to make it such that a change that doesn't conform to the schema fails checks and effectively cannot be merged. This way the clients team can ditch this on their end, simplifying their process.